src/plays/popup-component/components/popupCard.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import './styles.css';
+
+function popupCard({ children, popTitle, btnStyle }) {
```

Comment:
Hi, @lifeofsoumya 

The code is great, here are a few suggestions that will make it more readable and beautiful.

The component name `popupCard` should be changed to `PopupCard` to follow the common convention of starting component names with a capital letter.